wdl/transforms/biscayne/src/main/scala/wdl/transforms/biscayne/linking/expression/values/BiscayneValueEvaluators.scala

```diff
@@ -235,4 +235,46 @@ object BiscayneValueEvaluators {
         EvaluatedValue(WomArray(arr.value.map(v => WomString(v.valueString + suffix.value))), Seq.empty).validNel
       }
   }
+
+  /**
+   * Quote: Given an array of primitives, produce a new array in which all elements of the original are in quotes (").
+   * https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#-arraystring-quotearrayp
+   * input: Array[Primitive]
+   * output: Array[String]
+   */
+  implicit val quoteFunctionEvaluator: ValueEvaluator[Quote] = new ValueEvaluator[Quote] {
+    override def evaluateValue(a: Quote,
+                               inputs: Map[String, WomValue],
+                               ioFunctionSet: IoFunctionSet,
+                               forCommandInstantiationOptions: Option[ForCommandInstantiationOptions]
+    )(implicit expressionValueEvaluator: ValueEvaluator[ExpressionElement]): ErrorOr[EvaluatedValue[WomArray]] =
+      processValidatedSingleValue[WomArray, WomArray](
+        expressionValueEvaluator.evaluateValue(a.param, inputs, ioFunctionSet, forCommandInstantiationOptions)(
+          expressionValueEvaluator
+        )
+      ) { arr =>
+        EvaluatedValue(WomArray(arr.value.map(v => WomString("\"" + v.valueString + "\""))), Seq.empty).validNel
```

Comment:
Do you think we need to worry about escaping the value? The spec doesn't say anything about that, but as a user I would expect any quotes that already exist in the string to get escaped.